digitalocean/image/datasource_image_test.go

```diff
@@ -68,12 +71,12 @@ func TestAccDigitalOceanImage_PublicSlug(t *testing.T) {
 		CheckDestroy:      acceptance.TestAccCheckDigitalOceanDropletDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccCheckDigitalOceanImageConfig_slug("ubuntu-18-04-x64"),
+				Config: testAccCheckDigitalOceanImageConfig_slug("ubuntu-22-04-x64"),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
-						"data.digitalocean_image.foobar", "slug", "ubuntu-18-04-x64"),
+						"data.digitalocean_image.foobar", "slug", "ubuntu-22-04-x64"),
```

Comment:
It might be a good idea to add consts to the `acceptance` package for slugs like this so we can update them in one place.

Reply:
:100: